requirements/requirements_all.txt

```diff
@@ -27,7 +27,7 @@ ipykernel>=5.1.0
 ipython>=7.2.0
 codecov>=2.0.15
 pre-commit>=1.20.0
-shap>=0.36.0
+shap>=0.36.0,!=0.38.1
```

Comment:
Is it only 0.38.1, do we know if 0.38.2 will be ok?

Reply:
Well I was hoping to catch any new breaking releases in the daily cronjob.